dss/stepfunctions/__init__.py

```diff
@@ -5,6 +5,43 @@
 import boto3
 
 
+def step_functions_arn(state_machine_name_template: str) -> str:
+    """
+    The ARN of a state machine, with name derived from `state_machine_name_template`, with string formatting to
+    replace {stage} with the dss deployment stage.
+    :param state_machine_name_template:
+    :return:
+    """
+
+    sts_client = boto3.client("sts")
```

Comment:
Here and elsewhere, please use `dss.util.aws.clients.sts` (and `dss.util.aws.resources`), which avoids re-initializing the client every time you run this function.